tests/training/conftest.py

```diff
@@ -13,6 +13,7 @@
 #  limitations under the License.
 
 import argilla
```

Comment:
Maybe we can just remove this import and replace all the `argilla.` occurrences with `rg.`? WDYT @davidberenstein1957 

```suggestion
```

Reply:
Resolved in a324170bb895720290fd2a47f49890524d5bebdd, thanks for the review